test/RelayClient.test.ts

```diff
@@ -621,61 +668,16 @@ describe('RelayClient', function () {
         );
       });
 
-      it('should return request with given tokenGas', async function () {
-        const {
-          request: { tokenGas: expectedTokenGas },
-        } = FAKE_DEPLOY_REQUEST;
+      //FIXME check if we need to have a different approach
```

Comment:
Is this comment still valid? Which approach would you suggest and why do you find it wrong?

Reply:
the other approach was already validated, so its no longer needed to be fixed